lib/sql.js

```diff
@@ -75,17 +75,17 @@ SQL.prototype.execute = function(sql, vars, options, cb) {
   //TODO: why use POST if it's a write query?
   if (isGetRequest && !isWriteQuery(query)) {
     request.get({
-      url: this.sql_api_url, 
-      json: true, 
+      url: this.sql_api_url,
+      json: true,
       qs: params
-    }, processResponse);   
+    }, processResponse);
   } else {
     request.post({url: this.sql_api_url, json: true, form: params
-    }, processResponse);      
+    }, processResponse);
   }
 
   function processResponse(err, res, body) {
-    if(!err && res.statusCode==200) {
+    if(!err && res.statusCode === 200) {
```

Comment:
Always use triple equality. This is the kind of stuff that should be caught by a linter https://github.com/CartoDB/cartodb-nodejs/issues/39